src/core/Expander/Expander/Expander.md

```diff
@@ -25,8 +25,11 @@ import {
   <ExpanderTitle
     ariaOpenText="open expander"
     ariaCloseText="close expander"
+    toggleButtonAriaDescribedBy="checkbox-id"
   >
-    <Checkbox hintText="Checkbox hint text">Checkbox label</Checkbox>
+    <Checkbox hintText="Checkbox hint text" id="checkbox-id">
+      Checkbox label
+    </Checkbox>
   </ExpanderTitle>
   <ExpanderContent>Test expander</ExpanderContent>
 </Expander>;
```

Comment:
It seems that both examples using `ExpanderGroup` still uses the old, capitalised, prop names.

Reply:
Good catch! Damn I hate these md files without proper linter support.

Reply:
These should be now fixed!